pepr/utilities.py

```diff
@@ -51,3 +55,282 @@ def assign_record_ids_to_target_models(
             )
 
     return records_per_target_model + offset
+
+
+def filter_out_outlier(data, labels, filter_pars, save_path=None, load_pars=None):
+    """
+    Filter out potentially vulnerable samples.
+
+    The general idea is similar to (:meth:`pepr.privacy.gmia.DirectGmia`) to find
+    potential vulnerable records.
+
+    Steps:
+
+    1. Create mapping of records to reference models.
+    2. Train the reference models.
+    3. Generate intermediate models.
+    4. Extract high-level features.
+    5. Compute pairwise distances between high-level features.
+    6. Determine potential vulnerable records.
+    7. Remove potential vulnerable records.
```

Comment:
The function name is ``filter_out_outlier``so lets change the step descriptions accordingly:
6. Determine outlier records.
7. Remove outlier records from the dataset.

Probably we also want to rename the function to ``filter_outlier``, that's a little cleaner? :)

Reply:
Done in 30c739a